client/components/Insurance-Card/InsurancePage.vue

```diff
@@ -4,10 +4,13 @@
 <template>
   <main>
     <section>
-      <!-- <button>
-          Add contact
-        </button> -->
-      <CreateInsuranceForm />
+      <button v-if="!showForm" class="btn-primary" @click="showForm=true">
+        Add insurance
+      </button>
+      <button v-if="showForm" class="btn-primary" @click="showForm=false">
+        Add insurance
+      </button>
+      <CreateInsuranceForm v-if="showForm"/>
     </section>
```

Comment:
[nit] Not really related to this commit, but I noticed all three forms/cards have different titles:
- Create contact
- Add medication
- Add new insurance card

May be nice to standardize

Reply:
I think there's something nice about having separate names for the different components – it makes it a lot easier to read for a user who hasn't seen our code before

Reply:
Oh, I see where this is!